mail_environment/__openerp__.py

```diff
@@ -21,44 +21,12 @@
 
 {
     'name': 'Server env config for mail + fetchmail',
-    'version': '8.0.0.1.0',
+    'version': '9.0.1.0.0',
     'category': 'Tools',
-    'description': """
-Extend mail and fetch mail with server environment module.
-
-In config files, sections outgoing_mail and incoming_mails are default values
-for all Outgoing Mail Servers and Fetchmail Servers.
-For each server, you can (re)define values with a section named
-"outgoing_mail.resource_name" where resource_name is the name of your server.
-
-Exemple of config file :
-
-[outgoing_mail]
-smtp_host = smtp.myserver.com
-smtp_port = 587
-smtp_user =
-smtp_pass =
-smtp_encryption = ssl
-
-[outgoing_mail.openerp_smtp_server1]
-smtp_user = openerp
-smtp_pass = openerp
-
-[incoming_mail.openerp_pop_mail1]
-server = mail.myserver.com
-port = 110
-type = pop
-is_ssl = 0
-attach = 0
-original = 0
-user = openerp@myserver.com
-password = openerp
-
-
-    """,
+    'summary': 'configure mail servers with server_environment_files',
     'author': "Camptocamp,Odoo Community Association (OCA)",
     'license': 'AGPL-3',
-    'website': 'http://openerp.camptocamp.com',
+    'website': 'http://odoo-community.org',
     'depends': ['mail',
                 'fetchmail',
```

Comment:
I agree with @lmignon to be explicit regarding direct dependencies of a module. It allows to understand what this module extends (through XML-ID, data models and methods...), and reduces the scope of research of original resources.